plugins/snapshot.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: snapshot
+spec:
+  version: "v0.1.0"
+  platforms:
+
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.0.tar.gz
+    sha256: "369e4bbbe1afa76d5f363f7c123235ac556bd12989bc3fca286982f338086610"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-linux-amd64"
+      to:  "."
+    bin: "./kubectl-snapshot-linux-amd64"
+  - selector:
+      matchLabels:
+        os: linux
+        arch: 386
+    uri: https://github.com/fbrubbo/kubectl-snapshot/archive/v0.1.0.tar.gz
+    sha256: "369e4bbbe1afa76d5f363f7c123235ac556bd12989bc3fca286982f338086610"
+    files:
+    - from: "kubectl-snapshot-*/bin/kubectl-snapshot-linux-386"
+      to:  "."
+    bin: "./kubectl-snapshot-linux-386"
+
+  - selector:
+      matchLabels:
+        os: darwin
```

Comment:
actually it seems like this plugin works on all `linux` and `darwin` architectures, just do this: 

https://github.com/kubernetes-sigs/krew-index/blob/1adc818356eb63d643b0d544ff28e0805c51f34e/plugins/debug-shell.yaml#L13-L15

it seems like all platform specs seem identical.

Reply:
ignore that, looks like you actually compile separate binaries for each platform.